src/components/Viewer.js

```diff
@@ -32,6 +33,56 @@ const Viewer = ({ manifest }) => {
     initOpenSeadragon();
   }, [canvasImageResources]);
 
+  const calculateDownloadDimensions = () => {
+    let returnObj = {};
+
+    try {
+      let height,
+        width,
+        maxWidth = 3000,
+        canvasHeight = openSeadragonInstance.drawer.canvas.height,
+        canvasWidth = openSeadragonInstance.drawer.canvas.width,
+        proportionRatio = canvasHeight / canvasWidth;
+
+      if (canvasWidth > maxWidth) {
+        width = maxWidth;
+      }
+      width = canvasWidth > maxWidth ? maxWidth : canvasWidth;
```

Comment:
@adamjarling It looks like the last line of code, makes the if condition above redundant.

Reply:
Good catch.  I'll remove the extra conditional.